_layouts/docs.html

```diff
@@ -0,0 +1,126 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  {% include analytics.html %}
+  <title>microk8s - Fast, Light, Upstream Developer Kubernetes</title>
```

Comment:
We should probably make the `<title>` more specific - something like:

```suggestion

<title>
  {% if page.title %}
    {{ page.title }} | microk8s
  {% else %}
    microk8s - Fast, Light, Upstream Developer Kubernetes
  {% endif %}
</title>
```

(Practices could do with something like this too)

Reply:
Oho. I will try this out. Should work when using the Parsing information correctly in the markdown docs.

Reply:
Works great!